core/modules/system/system.admin.inc

```diff
@@ -1557,10 +1557,36 @@ function system_logging_settings($form, &$form_state) {
     ),
     '#description' => t('It is recommended that sites running on production environments do not display any errors.'),
   );
+  $form['watchdog_enabled_severity_levels'] = array(
+    '#type' => 'checkboxes',
+    '#title' => t("Enabled severity levels"),
```

Comment:
I'm confused by this label: "Enabled severity levels". It sounds like other severity levels won't work on my site, somehow. Why don't we just remove the description, and use that text as the label instead:

`Types of messages to log in the database.`

Watchdog is also drupal-insider language. The name of that module is now "Database logging" so we should probably refer to that as simply "the database log" or maybe "the database" is sufficient here.

Reply:
If Watchdog was ever a module, it was in Drupal 5 or before.  Since I started with Drupal 6, [watchdog](https://api.backdropcms.org/api/backdrop/core%21includes%21bootstrap.inc/function/watchdog/1) has been a core function used by modules to log significant events, patterned after the BSD syslog() function.

"Watchdog" is not only used by developers, but also by system administrators and site maintainers referring specifically to the logging generated by Drupal/Backdrop/Silkscreen, as opposed to logging from PHP, Nginx or Apache.  Eschewing the use of established nomenclature obscures the meaning of the setting.  Does "types of messages" mean "watchdog", or does it have a broader impact?

Avoiding "developer specific language" also erects barriers for the non-technical to join the ranks of developers and system administrators by putting off-limits the very language they need to even talk to the technical team, and for the technical team to explain issues with the site. If other terms are used (e.g., "Site logging"), then technical staff must maintain an internal dictionary mapping their terms to the plebeian versions.

"Watchdog" has a specific meaning, and that specific meaning is applicable here.  If the concern is the term will not make sense to some, then a short explanation in the `#description` tag should be added, possibly with a link to the API documentation or an entry in the Backdrop Administrator's Guide.

Consider instead setting `#title` to "Enabled Watchdog severity levels" and `#description` to "Backdrop and contributed modules use watchdog(link to API) to notify site owners and administrators of events on the site from expected activities (user registration or login) to errors (file not found, could not send email.)  More information is available in the Backdrop Administrator's Guide (link to logging)."

Reply:
> If Watchdog was ever a module, it was in Drupal 5 or before.

Yes, that's correct.
<img width="812" alt="screen shot 2019-01-01 at 1 00 06 pm" src="https://user-images.githubusercontent.com/397895/50576359-375beb00-0dc5-11e9-843e-1c6444953ae3.png">

> "Watchdog" has a specific meaning, and that specific meaning is applicable here.

"Watchdog" only as a specific meaning to those who understand Drupal/Backdrop. The intent here -- as with all of Backdrop's user interface -- is to make the software both usable and inviting for everyone, not only those with esoteric knowledge or specific job skills (like systems administration). 

My suggestion `Types of messages to log` will not confuse those who do not know what "Watchdog" means. If it applies to all internal logging (both DBlog and Syslog) no further disambiguation is necessary.

Reply:
> The intent here -- as with all of Backdrop's user interface -- is to make the software both usable and inviting for everyone, not only those with esoteric knowledge or specific job skills (like systems administration).

How does it stop being esoteric knowledge and become common knowledge?

Reply:
> How does it stop being esoteric knowledge and become common knowledge?

Maybe if WordPress, or even native desktop apps started using the same terminology?

Reply:
The "type of message" string is already in the #description.

Reply:
> The "type of message" string is already in the #description.

Great, so let's move that to the label, and remove the description entirely. Less clutter in the interface!